source/_integrations/spotify.markdown

```diff
@@ -36,9 +36,10 @@ an application at Spotify using the Spotify Developer website. This will
 provide you with the Spotify application credentials Home Assistant needs
 to allow you to log in with your Spotify account.
 
-1. Log in to the [Spotify Developer](https://developer.spotify.com) Dashboard.
-2. Visit the [My Applications](https://developer.spotify.com/my-applications/#!/applications) page.
-3. Click the **CREATE AN APP** button in the top right. Enter a name and
+1. If Spotify was previously integrated in your Home Assistant with _outdate_ credentials, it might be required to remove these old Spotify account credentials using the [Home Assistant Application Credentials dashboard](https://my.home-assistant.io/redirect/application_credentials/).
```

Comment:
```suggestion
1. If Spotify was previously integrated with your Home Assistant with _outdated_ credentials, it might be required to remove these old Spotify account credentials using the {% my application_credentials title="Home Assistant Application Credentials dashboard" %}.
```